Bff.AppHost/Bff.AppHost.csproj

```diff
@@ -1,17 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-  <Sdk Name="Aspire.AppHost.Sdk" Version="9.5.2" />
+<Project Sdk="Aspire.AppHost.Sdk/13.0.0">
```

Comment:
@damienbod I just aligned this with https://aspire.dev/whats-new/aspire-13/#-apphost-template-updates